### PR TITLE
Docs: Updating HTTP API endpoints with CLI equivalent links

### PR DESCRIPTION
**Pull request by ChipV223**
As the title suggests, this docs changes is to include a link to the CLI command from the equivalent HTTP API page

### CODE REVIEW

File: website/content/api-docs/acl/auth-methods.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/acl/auth-methods.mdx:37]: Please remove `https://www.consul.io` from all of these links (as annoying as it will be to update every line). That will allow for these links to work as intended anywhere the site is run (locally, a preview environment, or production). As is, these links will only ever point to production (even if working locally or in a preview/staging environment).

```suggestion
-> The corresponding CLI command is [`consul acl auth-method create`](/commands/acl/auth-method/create)
```

Before you apply this change, wait for a response on my question below to Karl and Adam. I want you to only need to touch each line once more rather than go back multiple times.

File: website/content/api-docs/acl/auth-methods.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/acl/auth-methods.mdx:37]: Thanks for noticing and correcting this inconsistency :)

File: website/content/api-docs/acl/auth-methods.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/acl/auth-methods.mdx:165]: @karl-cardenas-coding, @trujillo-adam : thoughts on whether:
- These sentences should end with a period?
- These sentences should be shown in an info box (blue background) or just as a normal sentence?

I don't think it warrants the info box highlight. (And I realize I contributed to this confusion by photoshopping in a blue highlight into my mock-ups ... next time I'll pick a different color to avoid potential confusion.) And if not in an info box, I think it should end in a period like a normal sentence. Want to confirm with you before asking @ChipV223 to make the edits though!

File: website/content/api-docs/acl/policies.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/acl/policies.mdx:163]: Nice - I like the use of the flag here!

File: website/content/api-docs/acl/tokens.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/acl/tokens.mdx:250]: Nice - this is something I was missing in my spreadsheet :) thanks

File: website/content/api-docs/connect/intentions.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/connect/intentions.mdx:461]: Copy-paste error
```suggestion
-> The corresponding CLI command is [`consul intention get`](https://www.consul.io/commands/intention/get)
```

Comment (jkirschner-hashicorp) [reply website/content/api-docs/connect/intentions.mdx:461]: Actually... for some reason, `consul intention list` is mentioned on the [intentions overview page](https://www.consul.io/commands/intention#basic-examples), but there doesn't appear to be a CLI command page entry for it?

Can you check whether `consul intention list` is a valid command? (If not, let me know, I just won't have time to check it for a few days.) We should probably add that command. And probably reference is from here. I don't actually think `consul intention get` is what we want here... it doesn't seem to list intentions.

File: website/content/api-docs/connect/intentions.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/connect/intentions.mdx:399]:
```suggestion
-> The corresponding CLI command is [`consul intention get`](https://www.consul.io/commands/intention/get)
```

File: website/content/api-docs/connect/intentions.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/connect/intentions.mdx:322]:
```suggestion
-> The corresponding CLI command is [`consul intention get`](https://www.consul.io/commands/intention/get)
```

File: website/content/api-docs/coordinate.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/coordinate.mdx:95]: Is the `#` there intentionally?

```suggestion
-> The corresponding CLI command is [`consul rtt`](https://www.consul.io/commands/rtt)
```

File: website/content/api-docs/coordinate.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/coordinate.mdx:41]: Is the `#` there intentionally?

```suggestion
-> The corresponding CLI command is [`consul rtt -wan`](https://www.consul.io/commands/rtt#wan)
```

**Review by jkirschner-hashicorp — CHANGES_REQUESTED** [review]
@ChipV223 : looks good! I'm excited to have these additions in the docs.

I spotted some minor corrections that (unfortunately) affect every line, but hopefully won't take too long to fix. I also have an open question for @karl-cardenas-coding / @trujillo-adam. Let's wait for them to respond before making any changes so that you just need to make one more pass through everything.

One other thing I noticed: **none of the admin partition commands are included in this change**. These don't exist in my spreadsheet because I created the spreadsheet before admin partitions were released. Should we include those in this PR? Or will that need to be done separately? I posted a similar question to @msiege2 on #11984.

File: website/content/api-docs/connect/intentions.mdx
Comment (ChipV223) [reply website/content/api-docs/connect/intentions.mdx:461]: @jkirschner-hashicorp So `consul intention list` is in fact a valid command but we have no corresponding CLI page

**Comment by ChipV223** [comment]
@jkirschner-hashicorp : The latest commit contains the links fixes as well as the addition of "." at the end of the sentence

File: website/content/api-docs/connect/intentions.mdx
Comment (jkirschner-hashicorp) [reply website/content/api-docs/connect/intentions.mdx]: I'll add a Github issue for that

File: website/content/api-docs/acl/auth-methods.mdx
Comment (ChipV223) [reply website/content/api-docs/acl/auth-methods.mdx]: Added a period during my later commits

File: website/content/api-docs/acl/auth-methods.mdx
Comment (ChipV223) [reply website/content/api-docs/acl/auth-methods.mdx]: Made the necessary changes in one of my later commits

File: website/content/api-docs/connect/intentions.mdx
Comment (ChipV223) [reply website/content/api-docs/connect/intentions.mdx]: Fixed in latest commit

File: website/content/api-docs/connect/intentions.mdx
Comment (ChipV223) [reply website/content/api-docs/connect/intentions.mdx]: Fixed in latest commit

File: website/content/api-docs/coordinate.mdx
Comment (ChipV223) [reply website/content/api-docs/coordinate.mdx]: Fixed in latest commit

File: website/content/api-docs/coordinate.mdx
Comment (ChipV223) [reply website/content/api-docs/coordinate.mdx]: Fixed in latest commit

File: website/content/api-docs/acl/auth-methods.mdx
Comment (jkirschner-hashicorp) [inline website/content/api-docs/acl/auth-methods.mdx:37]: After discussing with @karl-cardenas-coding and @trujillo-adam , we agree that we need to remove the info box treatment for these sentences (that gives them the blue background). Can you go through all these and remove the `-> ` at the front?

```suggestion
The corresponding CLI command is [`consul acl auth-method create`](/commands/acl/auth-method/create).
```

**Review by jkirschner-hashicorp — CHANGES_REQUESTED** [review]
One last thing to change and then we're good to go!

Admin partition stuff looks good, thanks.

File: website/content/api-docs/acl/auth-methods.mdx
Comment (jkirschner-hashicorp) [reply website/content/api-docs/acl/auth-methods.mdx]: I addressed my own feedback

**Review by jkirschner-hashicorp — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/555551.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 8a2256261329cb4a723a3e50341591eab8819c46 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 8a2256261329cb4a723a3e50341591eab8819c46 onto `release/1.11.x` succeeded!

**Pull request merged**
